MANIFEST.in

```diff
@@ -2,4 +2,3 @@ include tox.ini
 include README.rst
 include LICENSE
 include COPYING
-include globus_sdk/globus.cfg
```

Comment:
With this gone, we can probably remove the `pyinstaller` test, but I don't want to do too much at once. That will be follow-up.